src/scripts/docker-buildx.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+TAG=$(eval echo "$PARAM_TAG")
+SKIP_WHEN_TAGS_EXIST=$(eval echo "$PARAM_SKIP_WHEN_TAGS_EXIST")
+REPO=$(eval echo "${PARAM_REPO}")
+EXTRA_BUILD_ARGS=$(eval echo "${PARAM_EXTRA_BUILD_ARGS}")
+FILE_PATH=$(eval echo "${PARAM_PATH}")
+DOCKERFILE=$(eval echo "${PARAM_DOCKERFILE}")
+PROFILE_NAME=$(eval echo "${PARAM_PROFILE_NAME}")
+REGISTRY_ID=$(eval echo "\$${PARAM_REGISTRY_ID}")
+REGION=$(eval echo "${PARAM_REGION}")
+PLATFORM=$(eval echo "${PARAM_PLATFORM}")
+PUBLIC_REGISTRY=$(eval echo "${PARAM_PUBLIC_REGISTRY}")
+PUSH_IMAGE=$(eval echo "${PARAM_PUSH_IMAGE}")
+
+ACCOUNT_URL="${REGISTRY_ID}.dkr.ecr.${REGION}.amazonaws.com"
+number_of_tags_in_ecr=0
+docker_tag_args=""
+ECR_COMMAND="ecr"
+
+if [ "$PUBLIC_REGISTRY" == "1" ]; then
```

Comment:
Just making sure, I assume all public registries are only available in this region?

Reply:
Public repositories can be in other regions - i'm able to create a public repo in us-west-2 for example

Reply:
Good catch.  You just need to authenticate using `us-east-1 ` but you can push to other regions.  I removed line 21.